CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ if (POLICY CMP0026)
   cmake_policy(SET CMP0026 OLD)
 endif()
 
+# Has to be set before `project` as per documentation
+# https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_SYSROOT.html
+set(CMAKE_OSX_SYSROOT ${HERMES_APPLE_TARGET_PLATFORM})
```

Comment:
What does this do when building eg Android?

Reply:
No effect (as other "OSX" including flags)

Reply:
(haven't tested that, just referring to what cmake docs say. However, there were times when they didn't match the reality)